docs/visualization.rst

```diff
@@ -0,0 +1,340 @@
+Visualization interface
+=======================
+
+Launching the interface
```

Comment:
Add heading numbers.

1. Installation
2. Data Format Conventions and File Structure
3. Annotation Store Examples
4. Launching the Interface
5. General UI Control Options

Reply:
I wonder if we could have sphinx do this automatically. It would make edits easier and cleaner than manual numbering.